.github/workflows/label.yml

```diff
@@ -6,8 +6,9 @@
 # https://github.com/actions/labeler/blob/master/README.md
 
 name: PR Labeler
-on: [pull_request]
-
+on:
+  pull_request:
+    types: [opened, synchronize]
```

Comment:
What does this change do?

Reply:
it triggers the labeler only when the PR is opened or a commit is made. See https://github.com/banyan/auto-label/issues/23#issuecomment-596880205